### DISCUSSION TITLE
Help!

### DISCUSSION THREAD

**Discussion by rokko1244** [kookehs/bdo-marketplace#16] [General]
Hello there. I want to do this but can't succeed. Do you have a chance to share videos or explain the usage part step by step?

**Comment by rokko1244** [comment]
And 
![Screenshot_11](https://user-images.githubusercontent.com/76140992/102440808-fd2b9880-4031-11eb-91a5-739ddf49d41c.jpg)

**Comment by kookehs** [comment]
I don't have the bandwidth to create a video, but if you are stuck on a specific step I can elaborate more.
The image you've shared doesn't tell me much. I'm not sure if that script wraps around the executable or not.

You can always find the latest releases [here](https://github.com/kookehs/bdo-marketplace/releases) under `Assets`.

**Comment by rokko1244** [comment]
I have to do it with Java? Using visual studio?

**Comment by kookehs** [comment]
Nope.
Just download one of the executables.
Configure the `config.json` with your tokens and region.
Then create an `items.json` with IDs of items you want to retrieve.
Make sure they're all in the same folder and run the executable, then it'll output the filename specified in `config.json`.

**Comment by rokko1244** [comment]
Elinde .csv dosyası var mı? Paylaşır mısın?

**Comment by kookehs** [comment]
I do not have a CSV to share. Items you or I are interested will vary.

**Comment by rokko1244** [comment]
Creates a CSV but doesn't write anything in it.
![Screenshot_14](https://user-images.githubusercontent.com/76140992/102761797-70ae0c80-4388-11eb-83b7-42814070bffe.jpg)

**Reply by kookehs** [reply]
What does your `items.json` look like? Any error output from program?
An example one I use.
```json
[
	{
		"id": "719898",
		"name": "Fallen God's Armor",
		"grade": "1"
	},
	{
		"id": "7304",
		"name": "Strawberry",
		"grade": "1"
	},
	{
		"id": "9065",
		"name": "Milk",
		"grade": "0"
	},
	{
		"id": "705511",
		"name": "Manos Ring",
		"grade": "1"
	}
]

```
